recipes/crypto-rsa-pss-update/src/workflow.ts

```diff
@@ -217,9 +219,43 @@ function getCryptoBindings(root: SgRoot<JS>): string[] {
 		}
 	}
 
+	// Find promisified assignments that use the discovered bindings
+	const promisifiedBindings = getPromisifiedBindings(root, bindings);
+	bindings.push(...promisifiedBindings);
+
 	return bindings;
 }
 
+/**
+ * Find promisified wrappers that use crypto bindings discovered by resolveBindingPath
+ */
+function getPromisifiedBindings(root: SgRoot<JS>, existingBindings: string[]): string[] {
+	const promisifiedBindings: string[] = [];
+	const rootNode = root.root();
+
+	for (const binding of existingBindings) {
+		// Find: const someVar = util.promisify(crypto.generateKeyPair)
+		// or:   const someVar = util.promisify(generateKeyPair) 
+		const promisified = rootNode.findAll({
+			rule: {
+				pattern: `const $BINDING = util.promisify(${binding})`
+			}
+		});
```

Comment:
I think it not will works in scenario below

`const {promisify} = require("node:util")` 

probably you will need use`resolveBindingPath` for promisify

```suggestion
		// or:   const someVar = util.promisify(generateKeyPair) 
		const promisifyBind = resolveBindingPath(node, "$.promisify")
		const promisified = rootNode.findAll({
			rule: {
				pattern: `const $BINDING = ${promisifyBind}(${binding})`
			}
		});
```